README.md

```diff
@@ -15,4 +15,46 @@ The first parameter <PATH OF "auth.whisk.system"> is mandatory and the second pa
 <OPENWHISK_HOME> will be set to the parent directory of the openwhisk-catalog if it is not provided.
 If openwhisk-catalog directory is put in the openwhisk home directory, we are able to omit the
 second parameter. Otherwise, we need to specify the correct OPENWHISK_HOME either as the second
-parameter or the environment variable to make it run.
\ No newline at end of file
+parameter or the environment variable to make it run.
+
+# About the relationship of OpenWhisk and OpenWhisk-Catalog
```

Comment:
No need to use capital letter for the project name. Just simply use `openwhisk` and `openwhisk-catalog`